rotors_control/include/rotors_control/parameters_ros.h

```diff
@@ -12,7 +11,7 @@ template<typename T> inline void GetRosParameter(const ros::NodeHandle& nh,
                                                  const std::string& key,
                                                  const T& default_value,
                                                  T* value) {
-  CHECK_NOTNULL(value);
+  ROS_ERROR_COND(value == nullptr, "value is a nullptr.");
```

Comment:
True, but this has only effect if compiled in debug mode, I would rather have something written to the terminal, than just aborting in debug mode. Will change to `ROS_FATAL_COND` (which for some reason is not fatal), if this is ok with you?